src/build/github.rs

```diff
@@ -1,7 +1,7 @@
 //! This module is in charge of collecting contributions from GitHub.
 
 use crate::build::db;
-use anyhow::{bail, Context, Result};
+use anyhow::{bail, ensure, Context, Result};
```

Comment:
Didn't know about the `ensure` macro, love it! 😍

Reply:
I learn this one from `snafu` and just noticed that `anyhow` and `error-stack` also have such macros :D